src/debugletapi.js

```diff
@@ -225,12 +226,13 @@ DebugletApi.prototype.register_ = function(errorMessage, callback) {
 DebugletApi.prototype.listBreakpoints = function(callback) {
   var that = this;
   assert(that.debuggeeId_, 'should register first');
-  var url = API + '/debuggees/' + encodeURIComponent(that.debuggeeId_) +
-      '/breakpoints';
+  var query = { success_on_timeout: true };
   if (that.nextWaitToken_) {
-    url += '?waitToken=' + encodeURIComponent(that.nextWaitToken_);
+    query.waitToken = that.nextWaitToken;
   }
-  url += '?success_on_timeout=' + encodeURIComponent(true);
+
+  var url = API + '/debuggees/' + encodeURIComponent(that.debuggeeId_) +
```

Comment:
Are there any tests that exercise the `waitToken` case? They should have been failing while this was being incorrectly encoded.